File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L6/05/method.py
import sys

def beauty_reverse(num):
    while num > 0:
        print(num % 10, end='')
        num //= 100 * 10
    print()

def reverse():
    n = int(input())
    beauty_reverse(n)

def main():
    print("Enter an integer: ", end='')
    reverse()

if __name__ == "__main__":
    main()

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-05/plagiarized/L4/05/method.kt
package IR

import java.util.*

object method {
    //method reverse
    fun beautyReverse(num: Int) {
        var num = num
        while (num != 0) {
            val r = num % 10
            print(r)
            num /= 10
        }
        println()
    }

    fun reverse(n: Int) {
        beautyReverse(n)
    }

    //prog utama
    @JvmStatic
    fun main(args: Array<String>) {
        print("Enter an integer: ")
        val input = Scanner(System.`in`)
        val n = input.nextInt()
        //pamggil method
        reverse(n)
    }
}

